newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/ExpressionAutocompletionsDisplayer.js

```diff
@@ -225,6 +226,46 @@ const DisplayedBehaviorAutocompletion = React.forwardRef(
   }
 );
 
+type ExpressionDocumentationProps = {|
+  expressionMetadata: gdExpressionMetadata,
+  i18n: I18nType,
+  parameterRenderingService: ParameterRenderingServiceType,
+|};
+
+const ExpressionDocumentation = ({
+  expressionMetadata,
+  i18n,
+  parameterRenderingService,
+}: ExpressionDocumentationProps) => {
+  return (
+    <Column noMargin>
+      <Text style={defaultTextStyle} size="body2">
+        {expressionMetadata.getDescription()}
+      </Text>
+      {mapVector(
+        expressionMetadata.getParameters(),
+        (parameter, index) =>
+          !parameter.isCodeOnly() &&
+          // This filters parameters that are implicit because of the context.
+          // Free functions have an instanceContainer as first parameter,
+          // their first object parameter are kept.
+          (index !== 0 || parameter.getType() !== 'object') &&
+          (index !== 1 || parameter.getType() !== 'behavior') && (
+            <Text style={defaultTextStyle} size="body2">
```

Comment:
It might feel better organized visually if you remove the margin on the `Text` components and put all the components in a column.
Plus, you should have a warning because they don't have a key props. I think you can use the type + an index for the key?

Reply:
I'm not sure to follow.
The screenshot of the description is not up to date. It looks like this:

![image](https://user-images.githubusercontent.com/2611977/204002682-31e7e6a6-adee-43da-b9d1-1e5951a3e7d2.png)

Reply:
@AlexandreSi do you think something need to be changed?

Reply:
What I saw was that there is lot of space between each parameter (due to the `Text` component margins). I thought that maybe you could remove those margins (but maybe they will be too close to each other, I'll let you decide).

If, as a result, there's not enough margin between the description and the parameters, you can wrap the parameters in a `Column` component that brings its own margin.

Finally, the component that results from the map does not have a key, so React should be printing some errors/warnings in your JS console saying that they miss a key. So you could set the key such as:

```jsx
<Text style={defaultTextStyle} size="body2" key={`expression-metadata-parameter-${parameterIndex}`}>
```

Reply:
Ok, I added the key.
I like the spacing, it's not far from what VSC does.

![image](https://user-images.githubusercontent.com/2611977/204239713-64b189ce-6952-4aa6-a895-e61fc7a05e92.png)